submodules/eks/variables.tf

```diff
@@ -68,13 +74,16 @@ variable "default_node_groups" {
         {
           ami            = optional(string)
           instance_type  = optional(string, "g4dn.xlarge")
+          spot           = optional(bool, false)
           min_per_az     = optional(number, 0)
           max_per_az     = optional(number, 10)
           desired_per_az = optional(number, 0)
           labels = optional(map(string), {
             "dominodatalab.com/node-pool" = "default-gpu"
             "nvidia.com/gpu"              = true
           })
+          taints = optional(list(object({ key = string, value = optional(string), effect = string })), [])
```

Comment:
Does this need a default `nvidia.com/gpu` taint as well?

Reply:
yes, added.